### PR TITLE
Search engine/ft/new

### PR DESCRIPTION
**Pull request by NathanaelHermanto**
Screenshot 2022-01-09 at 20 28 40
created view for the results

### CODE REVIEW

**Comment by fabform** [comment]
Handy alternative list of [alternative search engines](https://fabform.io/a/alternative-search-engines)

File: packages/search-engine/src/direflow-components/search-engine/App.tsx [hunk 1]
```diff
@@ -1,136 +1,10 @@
...
-            <ResultView />
-        </div>
-      </div>
-      
-
-    </ReactiveBase>
-  </Styled>
+    <>
+      <SearchField  />
+    </>
```
Comment (sunanmau5) [inline packages/search-engine/src/direflow-components/search-engine/App.tsx]: you don't need the React Fragment wrapper around the <SearchField /> component, you can just do this
`const App: FC = () => <SearchField />`

**Review by sunanmau5 — APPROVED** [review]

**Comment by sunanmau5** [comment]
Merge conflict on `yarn.lock`

File: packages/search-engine/src/direflow-components/search-engine/components/ResultCard.tsx [hunk 2]
```diff
@@ -5,15 +5,16 @@ import {Avatar,
...
 const ResultCard: React.FunctionComponent<{ tweet: Tweet }> = ({ tweet }) => {
     return (
-    <Card sx={{ minWidth: 275 }}>
+    <Card sx={{ minWidth: 275, margin: '5px' }}>
         <CardContent>
-            <Box sx={{ display: 'inline-flex', alignItems: 'center', borderColor: '5px lightblue' }}>
-                <Avatar sx={{ marginRight: '0.5rem' }}>
-                    {tweet.partei.substring(0, 2)}
+            <Box sx={{ display: 'inline-flex', alignItems: 'center'}}>
+                <Avatar sx={{ marginRight: '0.5rem', fontSize: 15, bgcolor: alpha('#1976d2', 0.9) }}>
```
Comment (sunanmau5) [inline packages/search-engine/src/direflow-components/search-engine/components/ResultCard.tsx]: is it necessary to import another library for changing the color opacity of the `bgcolor` of the Avatar?
is it different if we do it like this?
`bgcolor: 'rgba(10, 46, 82, 0.9)'`

this is just a suggestion of course, you don't need to change it

File: packages/search-engine/src/direflow-components/search-engine/components/SearchField.tsx [hunk 3]
```diff
@@ -102,7 +102,18 @@ const SearchField: React.FunctionComponent = () => {
...
     const partyCheckBox = (
-        <Box sx={{ display: 'flex', flexDirection: 'row', ml: 3 }}>
+        <Box sx={{ 
+            display: 'flex', 
+            flexDirection: 'column', 
+            padding: '3px',
+            paddingLeft: '10px', 
+            border: 3,
+            borderRadius: 2, 
+            borderColor: `${alpha('#1976d2', 0.75)}`
```
Comment (sunanmau5) [inline packages/search-engine/src/direflow-components/search-engine/components/SearchField.tsx]: same thing here, is it different if we use the `rgba` value?
`bgcolor: 'rgba(10, 46, 82, 0.75)'`

**Review by sunanmau5 — APPROVED** [review]

File: packages/search-engine/src/direflow-components/search-engine/components/ResultCard.tsx [hunk 4]
```diff
@@ -13,7 +13,7 @@ const ResultCard: React.FunctionComponent<{ tweet: Tweet }> = ({ tweet }) => {
     <Card sx={{ minWidth: 275, margin: '5px' }}>
         <CardContent>
             <Box sx={{ display: 'inline-flex', alignItems: 'center'}}>
-                <Avatar sx={{ marginRight: '0.5rem', fontSize: 15, bgcolor: alpha('#1976d2', 0.9) }}>
+                <Avatar sx={{ marginRight: '0.5rem', fontSize: 15, bgcolor: 'rgba(25, 118, 210, 1)' }}>
```
Comment (sunanmau5) [inline packages/search-engine/src/direflow-components/search-engine/components/ResultCard.tsx:15]: don't forget to remove the `import { alpha }`

**Pull request merged**